tests/conftest.py

```diff
@@ -146,3 +147,14 @@ def static_path_provider(
     static_filename_provider: FilenameProvider,
 ):
     return static_path_provider_factory(static_filename_provider)
+
+
+@pytest.fixture
+def count_scan_trigger_info() -> TriggerInfo:
```

Comment:
```suggestion
def one_shot_trigger_info() -> TriggerInfo:
```

Reply:
Maybe? Makes it less implementation specific. It's just a TriggerInfo for 1 shot, without changing deadtime or livetime